CONTRIBUTING.rst

```diff
@@ -0,0 +1,76 @@
+Releasing
+=========
+
+Summary
+-------
+
+- Create a release branch in the main repository, not in a fork.
+- Prepare a commit for a release candidate, get an approving review, and tag the commit which will upload artifacts to PyPI.
```

Comment:
```suggestion
- Create a commit for a release candidate inside the release branch (see next for details
- Create a Pull Request for the release. The same branch and PR will be used for both the release candidate and  the final release.
- Get an approving review
- Once approved, tag the last commit in the branch as a release candidate.
- Push the tag and this trigger the upload of artifacts to PyPI.
```

Reply:
I Think the summary section is sufficiently short and well labeled to not need to reference ahead for details in the middle.  Also, other small adjustments.

```suggestion
- Create a commit for a release candidate inside the release branch.
- Create a Pull Request for the release. The same branch and PR will be used for both the release candidates and  the final release.
- Get an approving review.
- Once approved, tag the last commit in the branch as a release candidate.
- Push the tag.  This will trigger a build including the upload of artifacts to PyPI.
```